LX/src/main/java/heronarts/lx/LXSnapshotValue.java

```diff
@@ -0,0 +1,64 @@
+package heronarts.lx;
+
+import com.google.gson.JsonObject;
+
+import heronarts.lx.parameter.LXParameter;
+
+public class LXSnapshotValue extends LXComponent {
+
+  public Boolean isEnabled;
```

Comment:
Should be a BooleanParameter if this would be toggle-able in the UI and saved as a setting.